packages/eslint-plugin-gestalt/src/no-workflow-status-icon.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * @fileoverview Error on use a <Icon icon="workflow-status-[...]" size={16} />, and suggest use
+ * <Status /> component rather than <Icon />
+ */
+
+// @flow strict
+import { type ESLintRule } from './helpers/eslintFlowTypes.js';
+import { hasImport, isGestaltComponent, hasAttributes } from './helpers/eslintASTHelpers.js';
+
+const disallowedMatch = [
+  { icon: 'workflow-status-in-progress', color: 'success' },
+  { icon: 'workflow-status-ok', color: 'success' },
+  { icon: 'workflow-status-problem', color: 'error' },
+  { icon: 'workflow-status-warning', color: 'warning' },
+];
+
+export const errorMessage =
+  'Icons where `workflow-status-[...]` and matching color should be replaced for the Status Component. https://gestalt.pinterest.systems/status';
```

Comment:
This `workflow-status-[...]` Icon matches Status. Please, replaced with Status. https://gestalt.pinterest.systems/status';

Reply:
replaced for >>>> replaced with @EduhCosta